contracts/ContractRecovery.sol

```diff
@@ -72,7 +72,7 @@ contract ContractRecovery is CommonStorage {
 
   function exitRecoveryMode() public recovery auth {
     uint totalAuthorized = recoveryRolesCount;
-    // Don't double count the owner (if set);
+    // Don't double count the founder (if set);
```

Comment:
The comment has been changed, but the code has not...

Reply:
I think the code doesn't need to change. The comment needs to be reverted.

Reply:
This is the owner which is derived down from `DSAuth`